build.js

```diff
@@ -2,6 +2,8 @@
 
 const oryx = require('@spryker/oryx');
 const api = require('./lib');
-const configuration = api.getConfiguration(api.settings);
+const configurationPromise = api.getConfiguration(api.settings);
 
-oryx.build(configuration);
+configurationPromise
```

Comment:
can you change this into `api.getConfiguration(api.settings)` directly? Sorry, I fogot to mention before.